src/app/containers/Metadata/index.jsx

```diff
@@ -44,10 +44,10 @@ const MetadataContainer = ({ metadata, promo }) => {
   const timeFirstPublished = new Date(metadata.firstPublished).toISOString();
   const timeLastPublished = new Date(metadata.lastPublished).toISOString();
 
-  const canonicalLink = `${origin}/${service}/articles/${id}`;
+  const canonicalLink = `https://www.bbc.com/${service}/articles/${id}`;
   const canonicalLinkUK = `https://www.bbc.co.uk/${service}/articles/${id}`;
   const canonicalLinkNonUK = `https://www.bbc.com/${service}/articles/${id}`;
-  const ampLink = `${origin}/${service}/articles/${id}.amp`;
+  const ampLink = `https://www.bbc.com/${service}/articles/${id}.amp`;
```

Comment:
I just checked with @jimjohnsonrollings and the ampLink, which is used for the link: `<link rel="amphtml" href="" />` should vary depending on the origin of the user. 
```suggestion
  const ampLink = `${origin}/${service}/articles/${id}.amp`;
```
Only the canonical link should change.